src/main/java/adris/altoclef/tasks/GetToBlockTask.java

```diff
@@ -11,15 +12,28 @@
 
     private final BlockPos _position;
     private final boolean _preferStairs;
+    private final Dimension _dimension;
 
     public GetToBlockTask(BlockPos position, boolean preferStairs) {
```

Comment:
This is great but there's one really specific edge case, this tasks's `onFinish` is triggered when the bot is within its goal. This means if you happen to hit the right coordinate but the wrong dimension, it will fail.

Solution: Add
```java
    @Override
    public boolean isFinished(AltoClef mod) {
        return super.isFinished(mod) && _dimension == mod.getCurrentDimension();
    }
```

Reply:
I'll add this when I merge this in, no biggie